cache/client.go

```diff
@@ -21,6 +21,7 @@ const (
 	reasonMaxItemSize     = "max-item-size"
 	reasonAsyncBufferFull = "async-buffer-full"
 	reasonMalformedKey    = "malformed-key"
+	reasonConnectTimeout  = "connect-timeout"
 	reasonTimeout         = "timeout"
```

Comment:
WDYT about changing this to `requestRequestTimeout = "request-timeout"` to clarify it? We don't offer no breaking changes on metrics, so I wouldn't be much concerned changing the label value.

Reply:
The clearer, the better. 👌 

_changed @ 17f669b4474fb14db13d1458684169d0ae7179de_